marketplace/src/main/resources/package.xml

```diff
@@ -1,21 +1,27 @@
 <?xml version="1.0"?>
-<package type="addon" name="shibboleth-authentication" version="@VERSION@">
-  <title>Shibboleth Authentication</title>
-  <description>The Shibboleth Authentication addon allows to let your users authenticate against Shibboleth compliant identity management federation systems, providing a Single Sign On experience with other applications also plugged to the same system.</description>
-  <vendor>Nuxeo</vendor>
-  <classifier>Open Source</classifier>
-  <home-page>http://doc.nuxeo.com/display/NXDOC/Shibboleth+Authentication
-  </home-page>
-  <platforms>
-    <platform>server-@DISTRIB_VERSION@</platform>
-    <platform>server-@DISTRIB_VERSION@-*</platform>
-  </platforms>
-  <nuxeo-validation>nuxeo_certified</nuxeo-validation>
-  <license>LGPL</license>
-  <license-url>http://www.gnu.org/licenses/lgpl.html</license-url>
-  <supported>true</supported>
-  <production-state>production_ready</production-state>
-  <license>LGPL 2.1</license>
-  <license-url>http://www.gnu.org/licenses/lgpl-2.1.html</license-url>
-  <visibility>PUBLIC</visibility>
+<package type="addon" name="shibboleth-authentication" version="2.7.0-SNAPSHOT">
+    <title>Shibboleth Authentication</title>
+    <description>The Shibboleth Authentication addon allows to let your users authenticate against Shibboleth compliant
+        identity management federation systems, providing a Single Sign On experience with other applications also
+        plugged to the same system.
+    </description>
+    <vendor>Nuxeo</vendor>
+    <classifier>Open Source</classifier>
+    <home-page>http://doc.nuxeo.com/display/NXDOC/Shibboleth+Authentication
+    </home-page>
+    <platforms>
+        <platform>server-11.1-SNAPSHOT</platform>
```

Comment:
You must keep `@DISTRIB_VERSION@` and not fix it to a version, why did you change it?

Reply:
Yes, I agree, it was a mistake.